tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml

```diff
@@ -5,85 +5,19 @@
     xmlns:behaviors="using:DevHome.Common.Behaviors"
     xmlns:views="using:DevHome.Customization.Views" 
     xmlns:converters="using:CommunityToolkit.WinUI.Converters" 
-    xmlns:viewmodels="using:DevHome.Customization.ViewModels"
+    xmlns:viewmodels="using:DevHome.Customization.ViewModels" xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
     behaviors:NavigationViewHeaderBehavior.HeaderTemplate="{StaticResource BreadcrumbBarDataTemplate}"
     behaviors:NavigationViewHeaderBehavior.HeaderContext="{x:Bind ViewModel}">
 
-    <Page.Resources>
-        <converters:DoubleToVisibilityConverter x:Key="CountToVisibilityConverter" GreaterThan="0" FalseValue="Collapsed" TrueValue="Visible"/>
-
-        <DataTemplate x:Key="LocalRepositoryProviderButtonTemplate" x:DataType="viewmodels:FileExplorerSourceControlIntegrationViewModel">
-            <Button Content="{x:Bind ProviderName}" HorizontalAlignment="Stretch" Click="AddRepository_Click" Tag="{x:Bind}"/>
-        </DataTemplate>
-
-    </Page.Resources>
-
     <ScrollViewer VerticalAlignment="Top" VerticalScrollBarVisibility="Auto">
-        <Grid MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
+        <Grid MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}" HorizontalAlignment="Left">
```

Comment:
Why is the HorizontalAlignment needed here?

Reply:
Resolved per offline conversation of using StackPanel for page alignment. Thanks!